tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -14,29 +14,27 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# pylint: disable=import-self, invalid-name, unused-argument
+# pylint: disable=import-self, invalid-name, unused-argument, unused-variable, redefined-builtin
```

Comment:
can you say why you're disabling these extras?

Reply:
Just because there are too many lines to modify. LOL. Should I remove all unused variable？
![image](https://user-images.githubusercontent.com/32191045/180953331-16c11306-9b20-4b46-a10e-f9de07b1c936.png)

Reply:
ah yeah fair. if you have cycles, it would be great to remove these var--this lint check does tend to catch some bugs. at least though, for any pylint disable, let's add a comment explaining why we're adding it.